pkg/forwarder/forwarder.go

```diff
@@ -102,7 +102,8 @@ type Forwarder interface {
 
 // DefaultForwarder is in charge of receiving transaction payloads and sending them to Datadog backend over HTTP.
 type DefaultForwarder struct {
-	waitingPipe         chan Transaction
+	highPrio            chan Transaction
+	lowPrio             chan Transaction
```

Comment:
could you add a comment here describing what these 2 chans are respectively used for?

(If I understand correctly, one is used for the transactions that are being sent for the first time, the other for the transactions that are being retried)